docs/UserGuide.md

```diff
@@ -276,15 +276,16 @@ Format: gift
 
 ## Revenue
 
-Calculates the total amount of money earned from student who has paid
+Calculates the total amount of money earned from student who has paid/not paid
```

Comment:
It is not quite clear from this sentence that "not paid" refers to the total amount of money owed by students who have not yet paid. An alternate phrasing to be considered could be "Calculates the obtained revenue from students who have paid, or money to be owed from students who have not paid.".